.storybook/lucid-docs-addon/ChildComponents.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import _ from 'lodash';
+import marksy from 'marksy/components';
+import PropTypes from './PropTypes';
+import { stripIndent } from '../../src/docs/util';
+
+import SyntaxHighlighter, {
+	registerLanguage,
+} from 'react-syntax-highlighter/prism-light';
+import jsx from 'react-syntax-highlighter/languages/prism/jsx';
+import json from 'react-syntax-highlighter/languages/prism/json';
+import coy from 'react-syntax-highlighter/styles/prism/coy';
+
+const coyCustom = {
```

Comment:
is this the same as the other coyCustoms we've created? maybe we can move to a util method so it's shared